docs/content/configuration/creating_volumestorageclass.md

```diff
@@ -0,0 +1,69 @@
+# Creating a StorageClass
+
+Create a storage class YAML file in order to define the storage parameters, such as pool name, secret reference, `SpaceEfficiency`, and `fstype`.
+
+**Note:** If you are using the CSI Topology feature, in addition to the information and parameter definitions provided here, be sure to follow the steps in [Creating a StorageClass with topology awareness](creating_storageclass_topology_aware.md).
+
+Use the following procedure to create and apply the storage classes.
+
+**Note:** This procedure is applicable for both Kubernetes and Red Hat® OpenShift®. For Red Hat OpenShift, replace `kubectl` with `oc` in all relevant commands.
+
+Create a storage class YAML file, similar to the following `demo-storageclass.yaml` and update the storage parameters as needed.
+
+When configuring the file, be sure to use the same array secret and array secret namespace as defined in [Creating a Secret](creating_secret.md).
+
+Use the `SpaceEfficiency` parameters for each storage system, as defined in [the following table](#spaceefficiency). These values are not case-sensitive.
+
+#### `SpaceEfficiency` parameter definitions per storage system type
+
+|Storage system type|SpaceEfficiency parameter options|
+|-------------------|---------------------------------|
+|IBM FlashSystem® A9000 and A9000R|Always includes deduplication and compression. No need to specify during configuration.|
+|IBM Spectrum® Virtualize family|- `thick` (default value)<br />- `thin`<br />- `compressed`<br />- `dedup_thin` (creates volumes that are deduplicated with thin-provisioning)<br />- `dedup_compressed` (creates deduplicated and compressed volumes)<br /><br /> **Note:** <br />- The `deduplicated` value is deprecated. Use `dedup_compressed`, if possible. When used, `deduplicated` provide the same results as `dedup_compressed`.<br />- If not specified, the default value is `thick`.|
```

Comment:
`provide` -> `provides`

Reply:
typo fixed. thanks

Reply:
fixed in https://github.com/IBM/ibm-block-csi-driver/pull/466